app/assets/javascripts/discourse/app/lib/textarea-text-manipulation.js

```diff
@@ -1,6 +1,7 @@
 import { action } from "@ember/object";
```

Comment:
What @davidtaylorhq did is to keep the mixin around while we start using the new implementation in few places. This being said, this case seems easier and I can see you already handled the discourse-templates case, I also couldn't find another place to fix.